song-server/src/main/java/bio/overture/song/server/config/StorageConfig.java

```diff
@@ -40,9 +41,12 @@
   @Value("${score.url}")
   private String storageUrl;
 
+  @Value("${score.accessToken}")
+  private String scoreAccessToken;
+
   @Bean
   public StorageService storageService(){
-    return StorageService.createStorageService(new RestTemplate(),retryTemplate,storageUrl, validationService);
+    return createStorageService(new RestTemplate(), retryTemplate, storageUrl, validationService, "Bearer " + scoreAccessToken);
```

Comment:
create a static utility method that converts an access token to an authorization header value